src/desktop/apps/article/components/App.tsx

```diff
@@ -7,15 +7,45 @@ import { EditButton } from "desktop/apps/article/components/EditButton"
 import { ArticleLayout } from "./layouts/Article"
 import { data as sd } from "sharify"
 import { ArticleProps } from "@artsy/reaction/dist/Components/Publishing/Article"
+import { isProduction } from "lib/environment"
+import { articleAdCategory } from "../ad_category"
 
 export interface AppProps extends ArticleProps {
   templates?: {
     SuperArticleFooter: string
     SuperArticleHeader: string
   }
 }
-
+interface HtlbidProps {
+  cmd: any
+  setTargeting?: (a: string, b: string) => any
+}
 export class App extends React.Component<AppProps> {
+  componentDidMount() {
+    this.mountDFPScript()
```

Comment:
I don't think this should be called in `componentDidMount` but prepended in the `<head>` using `react-helmet`.

Reply:
We're not using `react-helmet` at the moment. `react-head` is used in reaction.